### PR TITLE
Add CNN prototype

### PR DESCRIPTION
**Pull request by vitillo**

### CODE REVIEW

File: profondo/cnn.py [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+
+    model.compile(loss=keras.losses.categorical_crossentropy,
+                  optimizer=keras.optimizers.Adam(), metrics=['accuracy'])
+
+    checkpointer = ModelCheckpoint(filepath=MODEL_FILE, verbose=1)
+    genre_weights = dict(enumerate(1 / y_train.mean(axis=0)))
+
+    callbacks = [checkpointer]
+    if args.logging:
+        print model.summary()
```
Comment (Dexterp37) [inline profondo/cnn.py]: Can you change this to print(model.summary()) so that works on Python 3 too?

File: profondo/cnn.py [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+    meta, images = load_data(args.input_dir)
+    x_train, y_train, x_test, y_test = train_test_split(meta, images)
+    try:
+        model = keras.models.load_model(MODEL_FILE)
+        logging.info("Checkpointed model loaded")
+    except Exception:
+        model = build_model(x_train, y_train)
+        logging.info("No checkpointed model found - building a new one")
+
+    model.compile(loss=keras.losses.categorical_crossentropy,
```
Comment (Dexterp37) [inline profondo/cnn.py]: Let's change this to loss=keras.losses.binary_crossentropy,

**Review by Dexterp37 — APPROVED** [review]
